src/component/BpmnVisualization.ts

```diff
@@ -56,4 +56,14 @@ export interface BpmnVisualizationOptions {
    * If set to `true`, activate panning i.e. the BPMN diagram is draggable and can be moved using the mouse.
    */
   mouseNavigationSupport: boolean;
+  zoomConfiguration?: ZoomConfiguration;
 }
+
+export type ZoomConfiguration = {
+  throttleDelay: number;
+  debounceDelay: number;
+  constraints?: {
+    maxZoom: number;
+    minZoom: number;
```

Comment:
:warning: this seems to be out of scope, see #732 for this feature + may behave inconsistently with the fit feature (see https://github.com/process-analytics/bpmn-visualization-js/issues/732#issuecomment-714769698)

Reply:
good point - I went little to far here :)
I will extract it